src/Parser/BloodDeathKnight/Modules/Items/T20_2pc.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class T20_2pc extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasBuff(SPELLS.BLOOD_T20_2PC_GRAVEWARDEN.id);
+    console.log('You have the 2pc');
```

Comment:
You can check which modules are enabled in the Development tab.

Reply:
fixed